Translated/Lobby/Concert/lvs_070010.csv

```diff
@@ -1,29 +1,29 @@
-lvs_070010_0010#0,"""The thousands of lives we fought for<br>The painful and unyielding fight"""
-lvs_070010_0015#0,"""Nightmares we face<br>Will we ever lay awake to see the sunlight?"""
-lvs_070010_0020#0,"""Bloodthirsty beasts on the frontal<br>Waiting for us to break as they mumble:"""
-lvs_070010_0025#0,"""\""Get lost, it's time<br>You cannot win every battle\"""""
-lvs_070010_0030#0,"""Stand by me, write your history<br>For the life that you forged is fractured and damaged"""
-lvs_070010_0035#0,"""Fan the fire; kindle the flame<br>Born in you is a warrior"""
-lvs_070010_0040#0,"""Be their barrier<br>Hold your spirits high! Keep up the fight!"""
-lvs_070010_0050#0,"""Your strength blesses us all<br>So shine like a beacon. Heed our call, my passionate friend"""
-lvs_070010_0055#0,"""I see the fire in your eyes burning bright<br>Rise up again and again"""
-lvs_070010_0060#0,"""We don't see to believe the bonds in the ones who carry us on<br>We'll never forget"""
-lvs_070010_0065#0,"""The tears and love we share<br>Hopeful prayers will shine like stars in the night eternally"""
-lvs_070010_0070#0,"""The words that could not be spoken<br>Are broken 'til they return to the earth"""
+lvs_070010_0010#0,"""The thousands of lives we fight for<br>We hold them in our tired arms"""
```

Comment:
This bit is something like "an incoming battle that the thousands strong embrace with open arms"

Reply:
"A thousand soldiers embracing<br>This battle in their open arms"

The Japanese has this as two lines, but one line would read better.

"A thousand soldiers embracing this battle in their open arms"